pkgs/development/tools/nimble/default.nix

```diff
@@ -3,13 +3,13 @@
 stdenv.mkDerivation rec {
   name = "nimble-${version}";
 
-  version = "0.7.4";
+  version = "0.7.8";
 
   src = fetchFromGitHub {
     owner = "nim-lang";
     repo = "nimble";
     rev = "v${version}";
-    sha256 = "1l477f1zlqpc738jg47pz599cwjasgy9jqdsplj3ywd12xfqpc96";
+    sha256 = "12znxzj1j5fflw2mkkrns9n7qg6sf207652zrdyf7h2jdyzzb73x";
   };
 
   buildInputs = [ nim ];
```

Comment:
this package does not build on darwin, because patchelf, does not exists there: https://hydra.nixos.org/build/41536593/nixlog/1